components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,50 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+@import '~@schibstedspain/sui-button-basic/lib/index';
+
+$mah-cover-basic: map-get($breakpoints, 's') !default;
+$fz-cover-basic-button: $fz-base !default;
+$ml-cover-basic-buttons: $m-base !default;
+$mt-cover-basic-buttons: $m-base !default;
+
+.sui-CoverBasic {
+  max-height: $mah-cover-basic;
+  position: relative;
+  width: 100%;
+
+  &.is-clickable {
+    cursor: hand;
+  }
+
+  &-gradient {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+  }
+
+  &-gradient:after {
+    background: linear-gradient(to bottom, rgba($c-black, 0) 25%, rgba($c-black, .75));
+    content: '';
+    display: inline-block;
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &-image {
+    display: block;
+    height: 100%;
+    object-fit: cover;
```

Comment:
Check out who may have a solution for you :-)
https://github.com/tonipinel/object-fit-polyfill
It's not a good solution at all. But if you really can't find a way to do it with css, it may inspire...